pyaqueduct/api.py

```diff
@@ -53,7 +53,7 @@ def create_experiment(
         experiment_data = self._client.create_experiment(title=title, description=description)
         return Experiment(
             client=self._client,
-            id=experiment_data.id,
+            experiment_id=experiment_data.id,
```

Comment:
Any reason for this particular change except this is a reserved keyword?

Reply:
That was the only reason.